### PR TITLE
Package rule for trellis-cli

### PR DESCRIPTION
**Pull request by bsirang**
Adds a rule to package trellis-cli into a tarball

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ RUN curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.10.1/baz
 RUN go get github.com/bazelbuild/buildtools/buildifier && \
     cp ~/go/bin/buildifier /usr/local/bin/
 
+# Default python binary to python3
+RUN ln -s /usr/bin/python3 /usr/bin/python
```
Comment (pickledgator) [inline docker/Dockerfile:22]: is this related to the CLI package?

Comment (bsirang) [reply docker/Dockerfile:22]: pkg_rules invoke python

**Review by pickledgator — APPROVED** [review]
